### ISSUE TITLE
[CI/G44] ddterm needs additional packages to run

### ISSUE THREAD

**Issue by Captn138** [ddterm/gnome-shell-extension-ddterm#549]
### GNOME Shell version

44.5

### Session type

Wayland (`echo $WAYLAND_DISPLAY` output is non-empty)

### Special configuration

- [ ] High DPI
- [X] More than one monitor

### ddterm version

44

### Linux distro

Arch Linux

### ddterm settings dump

[/]
ddterm-activate-hotkey=@as []
ddterm-toggle-hotkey=['F1']
palette=['rgb(23,20,33)', 'rgb(192,28,40)', 'rgb(38,162,105)', 'rgb(162,115,76)', 'rgb(18,72,139)', 'rgb(163,71,186)', 'rgb(42,161,179)', 'rgb(208,207,204)', 'rgb(94,92,100)', 'rgb(246,97,81)', 'rgb(51,218,122)', 'rgb(233,173,12)', 'rgb(42,123,222)', 'rgb(192,97,203)', 'rgb(51,199,222)', 'rgb(255,255,255)']
panel-icon-type='none'
shortcuts-enabled=true
tab-policy='automatic'

### Description

I installed the extension from the latest build available (https://github.com/ddterm/gnome-shell-extension-ddterm/actions/runs/6220919909) singe I am using Gnome 44 and when trying to toggle ddterm, I recieve the following notification.
No package is specified.
What could I be missing?
![image](https://github.com/ddterm/gnome-shell-extension-ddterm/assets/25801033/cd92a7f4-6c1a-4a18-8fa6-6b2692366aa5)

I also have tried different builds for Gnome 44 but they all behaved the same way.

**Comment by amezin** [comment]
Hm.

Could you run `journalctl -f` in any other terminal, and then try to start ddterm? And then copy `journalctl -f` output here.

**Comment by amezin** [comment]
Although this kind of message ("Please install the following packages:" but no list after it) shouldn't be possible. Are you sure the notification doesn't expand if you move the mouse over it?

**Comment by Captn138** [comment]
Hey, so I believed the notification wasn't expanded at first, but in compacted mode I could only see its first line. So the screenshot here is the actual whole notification.
I had to restart my computer for several reasons and when it did I noticed now that ddterm can actually start.
The issue now, is that it is totally transparent, even with the opacity setting not set to zero.
When I change the value, it refreshes and get a background, but when I toggle ddterm off and on again, it is back to being transparent.
![image](https://github.com/ddterm/gnome-shell-extension-ddterm/assets/25801033/865419fa-2bc4-447c-90dc-c6b8a0cedacc)

Just to be sure now I restarted my computer a couple of times but the behavior doesn't change.
